internal/stoppropaganda/websites.go

```diff
@@ -343,13 +345,39 @@ type Website struct {
 	Counter_code300 uint `json:"status_300"`
 	Counter_code400 uint `json:"status_400"`
 	Counter_code500 uint `json:"status_500"`
+}
 
-	host string
+func (ws *WebsiteStatus) IncreaseCounters(responseCode int) {
+	ws.Requests++
+	switch {
+	case responseCode < 200:
+		ws.Counter_code100++
+	case responseCode < 300:
+		ws.Counter_code200++
+	case responseCode < 400:
+		ws.Counter_code300++
+	case responseCode < 500:
+		ws.Counter_code400++
+	default:
+		ws.Counter_code500++
+	}
+}
+func (ws *WebsiteStatus) IncreaseCountersErr(errMsg string) {
+	ws.Requests++
+	if !strings.Contains(errMsg, customtcpdial.ErrTooFastDialSpam.Error()) {
```

Comment:
Don't you think this will be confusing for users?
```
        "https://mail.ru": {
            "requests": 2911,
            "errors": 15,
            "last_error_msg": "httpClient.Do: timeout",
            "status": "Running",
            "status_100": 0,
            "status_200": 0,
            "status_300": 0,
            "status_400": 0,
            "status_500": 0
        },
```
Should we use `client_errors` and `server_errors` instead, so counters match?

Reply:
Fixed with https://github.com/erkexzcx/stoppropaganda/pull/93/commits/e552a85487eb1bfb6749677da2ccd78df2dfd3c0